scripts/ecs-anywhere-install.sh

```diff
@@ -592,6 +602,74 @@ wait-agent-start() {
     fail
 }
 
+exec-setup() {
+    find-copy-certs-exec
+    download-ssm-binaries-exec
+}
+
+find-copy-certs-exec() {
+    # Reference for locations: https://golang.org/src/crypto/x509/root_linux.go
+    CERTS_FILES_LOCATIONS=("/etc/ssl/certs/ca-certificates.crt"
+      "/etc/pki/tls/certs/ca-bundle.crt"
+      "/etc/ssl/ca-bundle.pem"
+      "/etc/pki/tls/cacert.pem"
+      "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+      "/etc/ssl/cert.pem"
+    )
+    CERTS_PATH="/var/lib/ecs/deps/execute-command/certs"
+
+    echo "Copying certs for exec feature"
+
+    # Determine certs file
+    certs=""
+    if [ -n "$CERTS_FILE" ] && [ -f "$CERTS_FILE" ]; then
+        certs=$CERTS_FILE
+    else
+        if [ -n "$CERTS_FILE" ]; then
+            echo "Provided certs file does not exist, looking for certs in known possible locations"
+        fi
+        for f in "${CERTS_FILES_LOCATIONS[@]}"
+        do
```

Comment:
Q. Just a thought. Would it provide better UX if we printing out what certs have seen searched?
```
echo "Searching certs file `/etc/ssl/certs/ca-certificates.crt` "
...
echo "Could not find `/etc/ssl/certs/ca-certificates.crt`"
```

Reply:
Interesting point. I can see why some users might want to know these if the script doesn't find the certs. My thinking against doing that is that it would pollute the output a bit to log all of them, and that it's not super useful for the user to know given that these are often already present and easily able to be found just by looking at the script. Also, users can simply specify the location if the certs not found by default.

Reply:
I agree doing this will a kind of messing up output, and users can always provide a path if the certs cannot be found.